src/test/java/com/labuhn/minesweeper/ui/controller/MineCellCreatorTest.java

```diff
@@ -77,14 +81,17 @@ public void createCoveredFieldWithSurroundingMines(){
 
     @Test
     public void createUncoveredMineField(){
+        ImageView expectedImageView = new ImageView();
+        Mockito.when(iconProvider.createMineImage(25,25)).thenReturn(expectedImageView);
         Label cell = mineFieldCreator.createCell(new Cell(false, true, 4), null);
 
         assertThat(cell.getMinHeight()).isEqualTo(25);
         assertThat(cell.getMaxHeight()).isEqualTo(25);
         assertThat(cell.getMinWidth()).isEqualTo(25);
         assertThat(cell.getMaxHeight()).isEqualTo(25);
         assertThat(cell.getId()).isEqualTo("bomb");
-        assertThat(cell.getGraphic().getClass()).isEqualTo(ImageView.class);
+        Mockito.verify(iconProvider,Mockito.times(1)).createMineImage(25,25);
```

Comment:
try static import of Mockito. Makes statement more readable

Reply:
added static import